frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketItem/ticketItem.component.tsx

```diff
@@ -37,7 +37,7 @@ export const TicketItem = ({ ticket, onClick, selected }: TicketItemProps) => {
 	const { teamspace, project, containerOrFederation } = useParams<ViewerParams>();
 	const isFederation = modelIsFederation(containerOrFederation);
 	const template = TicketsHooksSelectors.selectTemplateById(containerOrFederation, ticket.type);
-	const hasIssueProperties = template?.config?.issueProperties;
+	const hasIssueProperties = ticket?.properties?.Priority;
```

Comment:
why the change in criteria?

Reply:
The config is only fetched when doing the singular fetchTemplate call which I have now moved into TicketDetails/NewTicket. Before I was running fetchTemplates and then looping over and running fetchTemplate, which was unnecessary.